app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -115,19 +111,20 @@ def proof_with_threatmetrix_if_needed(
         end
       end
 
+      # rubocop:disable Lint/UnusedMethodArgument
       def proof_residential_address_if_needed(
         applicant_pii:,
         timer:,
         double_address_verification: false,
         ipp_enrollment_in_progress: false
       )
-        return residential_address_unnecessary_result unless
-          ipp_enrollment_in_progress || double_address_verification
+        return residential_address_unnecessary_result unless ipp_enrollment_in_progress
 
         timer.time('residential address') do
           resolution_proofer.proof(applicant_pii)
         end
       end
+      # rubocop:enable Lint/UnusedMethodArgument
```

Comment:
is the linter requiring this b/c the default value of `ipp_enrollment_in_progress` is now `true`?

Reply:
Actually no. It's required because we're passing in double_address_verification as an argument, but not using that value in the method.

Reply:
ahh ok thank you for clarifying that!